_src/_components/download-source.pug

```diff
@@ -9,11 +9,11 @@
                 th.downtable5 Source
             tbody
                 +downloadRow({
-                    ver: "3.0.0 (Alpha 5)",
+                    ver: "3.0.0 (Beta 1)",
                     date: "2022-06-15",
```

Comment:
Need to change the date as well. Apache Ignite 3 beta1 was released in 17/11/2022 ("2022-17-11")